test/sandbox/sandbox.go

```diff
@@ -607,7 +607,6 @@ func buildTree(t testing.TB, root *config.Root, layout []string) {
 		assert.NoError(t, err)
 
 		cfg.Stack = &hcl.Stack{}
-		cfg.Terramate = &hcl.Terramate{}
```

Comment:
minor and unrelated, but when I did `--log-level=warning` in the e2etests I saw the warning message of `terramate` block defined in non-root directory. The sandbox was still generating an empty terramate block when creating test stacks.